tests/dialects/test_scf.py

```diff
@@ -1,5 +1,10 @@
-import pytest
+"""
+Test the usage of scf dialect.
+"""
+
 from typing import cast
+import pytest
```

Comment:
Better import pytest first for homegeneity

Reply:
my linter complains

```
# W: standard import "from typing import cast" should be placed before "import pytest"
```